estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models,fields
+
+class PropertyTag(models.Model):
```

Comment:
```suggestion
from odoo import models,fields


class PropertyTag(models.Model):
```
We usually leave 2  empty line before class definition